task/orka-full/0.1/orka-full.yaml

```diff
@@ -0,0 +1,159 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: orka-full
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.16.0"
+spec:
+  description: >-
+    With this set of Tasks, you can use your Orka environment
```

Comment:
nit: `With this set of Tasks` -> `With this Task`?

Reply:
Good catch